engine.fhir/pom.xml

```diff
@@ -22,8 +20,26 @@
         <dependency>
             <groupId>org.opencds.cqf.cql</groupId>
             <artifactId>engine</artifactId>
-            <version>1.5.8-SNAPSHOT</version>
+            <version>${project.version}</version>
```

Comment:
This requires the use of the maven-flatten plugin to produce deployable poms. The maven-flatten plugin breaks the "dirty" detection on maven builds which extends build times and sometimes breaks code coverage. Rather than use this parameter, or have to set the versions manually, there's a plugin for maven to do it for you automatically. We've added a scripts to help:

`./scripts/setversion.sh 2.0.0-SNAPSHOT`